src/models/models.ts

```diff
@@ -0,0 +1,52 @@
+export enum Status {
+  Available = 'available for checkout',
+  InStorage = 'in storage',
+  NeedsRepair = 'needs repair',
+}
```

Comment:
Typically with enums like this, we'll capitalize the enum itself, `Status` and then the variables inside will be all caps like so:

```suggestion
export enum Status {
  AVAILABLE = 'available for checkout',
  CHECKED_OUT = 'checked out'
  IN_STORAGE = 'in storage',
  NEEDS_REPAIR = 'needs repair',
  NOT_AVAILABLE = 'not available',
}
```

added a couple more statuses we could possibly run into

BTW, not sure if you know this or not, but you can click "commit suggestion" and it will auto add the suggested code without you having to open vs code, etc